doctrine/orm-toolbox.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "registrar": [
+        {
+            "language": "php",
+            "provider": "doctrine.models",
+            "signature": [
+                "Doctrine\\ORM\\EntityManagerInterface::copy",
```

Comment:
Why you are duplicate this methods in short notation? It is already described in `signatures` section